### ISSUE TITLE
RegNet hyperparameters questions

### ISSUE THREAD

**Issue by sgunasekar** [kuangliu/pytorch-cifar#139]
Hi ,

I have been trying to reproduce the RegNet results, but unable to go past 91 on CIFAR, if you still have access to it would you mind sharing the learning rate you used? 

Also, I noticed that for RegNetY_400MF, you use the same configuration as RegNetX_400MF with se_ratio=0.25, however, the official implementations uses a different configuration albeit the design was based on performance on imagenet -- was this change from the paper intentional based on performance on CIFAR?

Thanks for maintaining the repo.
